settings.tcl

```diff
@@ -111,6 +113,7 @@ array set ::iconik_settings {
     saver_dir {/saver}
 
     show_water_level_indicator 0
+    show_ml_instead_of_water_level 0
```

Comment:
Please enable this by default

Reply:
Sure, will change. I just did not want to change default behavior in the first without talking to you as the owner of the skin.